reference_simulations/README.md

```diff
@@ -30,6 +30,10 @@ Manager (SLURM).
         Given the paths to the latest output files (uvh5), this will compare the data in those files
         to the corresponding files in `latest_ref_data`.
 
+ - get_gleam.py
+        A script to download the GLEAM extragalactic source catalog from Vizier, using `astroquery`,
+        and save it to a VOTable file. GLEAM, treated as flat-spectrum, is used for several reference simulations.
```

Comment:
This could be very useful even to users that are not worried about the reference sims. Maybe we could document it somewhere else as well?

Reply:
This only downloads the `Fintwide` flux, as used by the first round of reference sims. Since you've added a script to `pyradiosky` that downloads all subbands for GLEAM, we may not want to promote this one beyond its use for reference sims.

Reply:
Yeah, I figured that out when I had James use it to get GLEAM on Friday. He wanted to look at the spectral index, which didn't work because the columns were missing. That's actually why I wrote the script in `pyradiosky`.